include/fdreadoutlibs/wib2/tpg/RegisterToChannelNumber.hpp

```diff
@@ -25,7 +25,7 @@ namespace swtpg_wib2 {
 struct RegisterChannelMap
 {
   // TODO: Make these the right size
-  uint channel[256];
+  uint channel[128];
```

Comment:
How did the this change? Should be replaced by const expression.

Reply:
Fixed using the const expression from the TPGConstants file. I noticed we were using arrays of 256 but in practice, we always use only 128. Now it's not hardcoded anymore.